src/partials.ts

```diff
@@ -6,23 +6,25 @@ import { resolve, parse } from 'path';
 /**
  * Registers each HTML file in a directory as Handlebars partial
  */
-export async function registerPartials(directoryPath: string): Promise<void> {
-  let dir: Dir;
+export async function registerPartials(directoryPath: Array<string>): Promise<void> {
```

Comment:
We'll want to allow this to be one or more paths

```suggestion
export async function registerPartials(directoryPath: string | Array<string>): Promise<void> {
```

Then inside the function, we can normalize it to always be an array by wrapping the single string if the user provided it that way